analysis_notebooks/Geography Analysis.ipynb

```diff
@@ -731,13 +776,52 @@
     "                subregion_pvalues_over[p] = tag_info\n",
     "            else:\n",
     "                subregion_pvalues_under[p] = tag_info\n",
+    "                \n",
+    "elif (dataset.geography_info_type == \"STRING_FORMATTED_LABEL\" and dataset.geography_label_string_type == \"REGION_LABEL\"):\n",
+    "    print(\"Geo_tag work for region label formatted dataset\")\n",
+    "    if not os.path.exists(\"results/{0}/6\".format(folder_name)):\n",
+    "        os.mkdir(\"results/{0}/6\".format(folder_name))\n",
+    "    info_stats = pickle.load(open(\"results/{}/geo_tag_region.pkl\".format(folder_name), \"rb\")) #20GB\n",
+    "    region_tags = info_stats['region_tags']\n",
+    "    tag_to_region_features = info_stats['tag_to_region_features']\n",
+    "\n",
+    "    categories = dataset.categories\n",
+    "    total_counts = np.zeros(len(categories))\n",
+    "\n",
+    "    for region, counts in region_tags.items():\n",
+    "        total_counts = np.add(total_counts, counts)\n",
     "\n",
-    "if not is_country:\n",
+    "    total_counts = total_counts.astype(int)\n",
+    "    sum_total_counts = int(np.sum(total_counts))\n",
+    "\n",
+    "    if not os.path.exists('checkpoints/{}/6_a.pkl'.format(folder_name)):\n",
```

Comment:
same here. if prerun_geo doesn't have these changes either, please add in

Reply:
changed 6_a to new naming convention ("geo_tag_6a" ). Also added new changes to prerun_geo